mcp/include/xsc_pointing.h

```diff
@@ -31,6 +31,8 @@
 #include <stdint.h>
 #include "pointing_struct.h"
 
+#define AVG_ARCSEC_PER_PX 6.22 // from blastcam:star camera output
```

Comment:
This updates the old star camera hardcoded value with a new one derived from Ian and Evan's test output.